htdocs/mrp/mo_list.php

```diff
@@ -207,6 +217,76 @@
 	$objectlabel = 'Mo';
 	$uploaddir = $conf->mrp->dir_output;
 	include DOL_DOCUMENT_ROOT.'/core/actions_massactions.inc.php';
+	$objMo = new Mo($db);
+
+	if($action == 'confirm_cancel') {
```

Comment:
You should install the precommit github hook so this king of CI error are automatically fixed.